Implemented features from the zydis header

Colton1skees opened the pull request:
- Added the [ZydisFeature enum](https://github.com/zyantific/zydis/blob/5af06d64432aaa3f6af3cd3e120eefa061b790ab/include/Zydis/Zydis.h#L112)
- Imported all functions from the Zydis,h header

Commit:
zydisWrapper

flobernd commented on Zyantific.Zydis/Native/Zydis.cs:
```diff
@@ -1,7 +1,34 @@
+    }
+
+    [StructLayout(LayoutKind.Sequential)]
+#if ZYDIS_UNSAFE
```
The unsafe block is not neccessary, if the class does not contain any unsafe code.

flobernd commented on Zyantific.Zydis/Native/Zydis.cs:
```diff
@@ -1,7 +1,34 @@
+
+        [DllImport(nameof(Zyantific.Zydis), ExactSpelling = true,
+            EntryPoint = ImportNameGetVersion)]
+        public static extern ZyanStatus FeatureEnabled(ZydisFeature feature);
```
Please try to use the same names like in the C code (wirhout prefix is correct). Ist should be IsFeatureEnabled in this case.

Commit:
remove unnecessary unsafe block and naming issues

Commit:
remove unnecessary unsafe block and naming issues

remove unnecessary attribute

remove spacing issue

Implemented several zydis structs

Revert "Implemented several zydis structs"

This reverts commit b658dbda45daf9bbc8bacb34c56d11cd5878a8fb.
